src/executors/ci-base.yml

```diff
@@ -1,3 +1,10 @@
+parameters:
+  tag:
+    type: string
+    default: latest
```

Comment:
`latest` isn't a tag for this image. A good fit here is likely the "stable" tag. I would also add a description mentioning the tag schema which is `yyyy.mm` or also `edge`.

Or maybe alternatively, link to the readme for the repo which explains all of this.

Reply:
@felicianotech According to the README in the repository it is :) also https://hub.docker.com/layers/cibuilds/base/latest/images/sha256-cc2ed7dd6551a95bd84144a95d06286c980c1394a37b3b0f010758b6dd68aadc?context=explore

Reply:
Where do you see it in the readme? I just checked, I don't see it.

As for the tag on DockerHub, that's a mistake and I'll need to fix that then.